vscode/package.json

```diff
@@ -75,6 +75,56 @@
         "title": "Run Analysis",
         "category": "Konveyor",
         "icon": "$(play}"
+      },
+      {
+        "command": "konveyor.diffView.applyAll",
+        "title": "Apply All",
+        "category": "Resolutions",
+        "icon": {
+          "light": "resources/icons/light/check.svg",
+          "dark": "resources/icons/dark/check.svg"
+        }
+      },
+      {
+        "command": "konveyor.diffView.revertAll",
+        "title": "Discard local changes to Resolutions",
+        "category": "Resolutions",
+        "icon": {
+          "light": "resources/icons/light/discard.svg",
+          "dark": "resources/icons/dark/discard.svg"
+        }
+      },
+      {
+        "command": "konveyor.diffView.applyFile",
+        "title": "Apply",
+        "icon": {
+          "light": "resources/icons/light/check.svg",
+          "dark": "resources/icons/dark/check.svg"
+        }
+      },
+      {
+        "command": "konveyor.diffView.revertFile",
+        "title": "Discard local changes to Resolutions",
+        "icon": {
+          "light": "resources/icons/light/discard.svg",
+          "dark": "resources/icons/dark/discard.svg"
+        }
+      },
+      {
+        "command": "konveyor.diffView.copyDiff",
+        "title": "Copy Diff"
+      },
+      {
+        "command": "konveyor.diffView.copyPath",
+        "title": "Copy Path"
+      },
+      {
+        "command": "konveyor.diffView.next",
+        "title": "Go to Next File"
+      },
+      {
+        "command": "konveyor.diffView.prev",
+        "title": "Go to Previous File"
```

Comment:
What I was expecting as an implementation of #70, was to have a new command like `konveyor.viewFix` that would be called by the webview when a user clicks a certain button/target (like the filename). This command would the actual change to be showed to the user in context as either a unified/inline or a split diff. That diff can either be "accept"ed (maybe a `konveyor.acceptFix`) or "reject"ed (`konveyor.rejectFix`).

I reject the idea that our extension needs to follow the example of a SCM.

All I want to see is the ability to see a specific fix in it's context.

Reply:
I'll add a `konveyor.viewFix` command (the interface/name may  change).

>That diff can either be "accept"ed (maybe a konveyor.acceptFix) or "reject"ed (konveyor.rejectFix).
>
Note that both the current code (left side of the diff) and resolutions (right side of the diff) might be edited. 
The final code change will include those local changes (once we implement that action).

Reply:
Added `konveyor.diffView.viewFix` command.